src/redux/reducer.js

```
@@ -20,13 +20,29 @@ export const loginUser = createAsyncThunk(
 )
 
+export const createUser = createAsyncThunk(
+  'appMain/createUser',
+  async (userData, { rejectWithValue, dispatch }) => {
+    try {
+      const { data } = await AuthService.signup(userData)
+      console.log(data)
+      await SecureStore.setItemAsync('accessToken', data.accessToken)
+      dispatch(setUser(data.accessToken))
+    } catch (error) {
+      if (error.response && error.response.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)